client/src/components/list-paginator/list-paginator.html

```diff
@@ -0,0 +1,19 @@
+<div class="pagination">
+    <div data-bind="click: goToFirstPage" class="pagination__page">First</div>
+    <div data-bind="click: goToPreviousPage" class="pagination__page">Prev</div>
+    <div data-bind="text: (currentPage() + 1) + '/' + pageCount()" class="pagination__page"></div>
```

Comment:
Not only is this calculation not to be living here, but also is duplicated below, which is totally unacceptable.